[refactor] #4029, #4105, #4136: refactor CLIs (#4153)
hyperledger/iroha: README.md

### Configuration

There is a set of configuration parameters that could be passed either through a configuration file or environment variables.

```shell

> # will look up for `config.json` or `config.json5`, without failing if files are not found

Reviewer: ```suggestion
# look for `config.json` or `config.json5` (won't fail if files are not found)
```

### Storage

> Iroha stores blocks and snapshots in the `storage` directory, which is created automatically by Iroha in the working directory of the peer. If `kura.block_store_path` is overwritten in the config file, it will be resolved relative to the config file location.

Reviewer: ```suggestion
Iroha stores blocks and snapshots in the `storage` directory, which is created automatically by Iroha in the working directory of the peer. If `kura.block_store_path` is specified in the config file, it overwrites the default one and is resolved relative to the config file location.
```
